src/cmd/linuxkit/push_vcenter.go

```diff
@@ -0,0 +1,304 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25/soap"
+	"github.com/vmware/govmomi/vim25/types"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type vmConfig struct {
+	vCenterURL  *string
+	dsName      *string
+	networkName *string
+	vSphereHost *string
+
+	vmName     *string
+	iso        *bool
+	disk       *bool
+	isoPath    *string
+	diskPath   *string
+	persistent *int64
+	vCpus      *int
+	mem        *int64
+}
+
+func exit(err error) {
+	log.Fatalf("Error: %s\n", err)
+	os.Exit(1)
```

Comment:
I'm not in favor of having an `exit` function for two reasons:
- it's not needed because there's log fatal
- looks like a language's primitive and future readers might misunderstand it